code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -94,8 +94,6 @@
 
 /obj/item/melee/flyswatter/attack(mob/living/M, mob/living/user, def_zone)
 	. = ..()
```

Comment:
Nearly no mob returns any value here currently so all guchi to delete the check for now